packages/env/src/utils.js

```diff
@@ -74,3 +74,14 @@ export function stripQuotesAndSpaces(line) {
     return strippedLine;
   } else { return null; };
 }
+
+// convert tags comma-separated-names to array of objects for POST request
+export function tagsList(tags) {
```

Comment:
Not a correct place to keep this function - this page is for environment related functions - you can move it to client package as we only need it there - but later we can move to a `@percy/common` like package